target_chains/sui/scripts/pyth/create_all_price_feeds.ts

```diff
@@ -1,118 +1,63 @@
-/// We build a programmable txn to create all price feeds.
-import dotenv from "dotenv";
-import axios from "axios";
 import { PriceServiceConnection } from "@pythnetwork/price-service-client";
 import * as _ from "lodash";
-import {
-  RawSigner,
-  SUI_CLOCK_OBJECT_ID,
-  TransactionBlock,
-  JsonRpcProvider,
-  Ed25519Keypair,
-  Connection,
-} from "@mysten/sui.js";
+import { Ed25519Keypair } from "@mysten/sui.js";
 
-dotenv.config({ path: "~/.env" });
+import { DefaultStore, SuiContract } from "@pythnetwork/pyth-contract-manager";
 
-import { REGISTRY, NETWORK } from "../registry";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
 
-// ================== Network dependent settings ==================
-let network = NETWORK.MAINNET; // <= NOTE: Update this when changing network
-const walletPrivateKey = process.env.SUI_MAINNET; // <= NOTE: Update this when changing network
-const price_feed_id_url = "https://xc-mainnet.pyth.network/api/price_feed_ids"; // <= NOTE: Update this when changing network
-const connection = new PriceServiceConnection(
-  "https://xc-mainnet.pyth.network", // <= NOTE: Update this when changing network
-  {
-    priceFeedRequestConfig: {
-      binary: true,
+const parser = yargs(hideBin(process.argv))
+  .scriptName("create_all_price_feeds.ts")
+  .usage("Usage: $0 --contract <contract_id> --private-key <private_key>")
```

Comment:
perhaps rename this to `package_id`? In the Sui docs they call it package instead of contract. The terminology seems standardized in their ecosystem

Reply:
Well, this is actually not the package_id it's a combination of the chain and state_id as defined in the contract manager. I agree that we should document it better.